runtime-modules/working-group/src/lib.rs

```diff
@@ -1329,11 +1365,21 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
             )
             .map_err(|e| DispatchError::Other(e))
     }
+
+    fn ensure_worker_role_storage_text_is_valid(text: &[u8]) -> DispatchResult {
+        Self::worker_storage_text()
+            .ensure_valid(
+                text.len(),
+                Error::<T, I>::WorkerStorageTextTooShort.into(),
+                Error::<T, I>::WorkerStorageTextTooLong.into(),
+            )
+            .map_err(|e| DispatchError::Other(e))
+    }
 }
 
 /// Creates default text constraint.
 pub fn default_text_constraint() -> InputValidationLengthConstraint {
-    InputValidationLengthConstraint::new(1, 1024)
+    InputValidationLengthConstraint::new(1, 2048)
```

Comment:
I think we should keep this default, and have a separate "default" for the storage field if we want to allow 0 length.

Reply:
Good idea, https://github.com/Joystream/joystream/pull/2180/commits/9f1d0ce17addd360318b008408e4f5516656d6ec